src/momento/cache_operation_types.py

```diff
@@ -130,54 +138,97 @@ def values_as_bytes(self) -> List[Optional[bytes]]:
     def to_list(self) -> List[CacheGetResponse]:
         return self._responses
 
+    def __str__(self) -> str:
+        return self.__repr__()
+
     def __repr__(self) -> str:
         return f"CacheMultiGetResponse(responses={self._responses!r})"
 
 
 class CacheDeleteResponse:
-    def __init__(self, grpc_create_cache_response: Any):  # type: ignore[misc]
+    def __init__(self) -> None:
         pass
 
+    @staticmethod
+    def from_grpc_response(grpc_create_cache_response: Any) -> "CacheDeleteResponse":  # type: ignore[misc]
```

Comment:
copy pasta issue on this? `grpc_create_cache_response` should be `grpc_delete_cache_response`

Reply:
Good catch

Reply:
I opted to just delete these methods altogether. They aren't serving any purpose for now.